app/views/schools/contacts/new.html.erb

```diff
@@ -2,18 +2,18 @@
 
 <% if @contact.user == current_user %>
   <h2>
-    Sign up for email or text alerts
+    <%= t('schools.contacts.new.sign_up_for_email_or_text_alerts') %>
   </h2>
   <%= render 'shared/contact_explainer' %>
 <% else %>
   <h2>
-    Enable alerts <%= "for #{@contact.user.name}" if @contact.user %>
+    <%= t('schools.contacts.new.enable_alerts_for', user_name: @contact.user.name) if @contact.user %>
```

Comment:
Same as above, the logic is not the same as the original. "Enable alerts" also won't be displayed if there is no @contact.user. Not sure if this is intentional or not?